homeassistant/bootstrap.py

```diff
@@ -402,7 +402,13 @@ def log_exception(ex, domain, config):
                    .format(ex.path[-1], domain, domain,
                            '->'.join('%s' % m for m in ex.path))
     else:
-        message += humanize_error(config, ex)
+        try:
+            message += humanize_error(config, ex)
+        except TypeError:
+            # Shorten ex.path - can be caused by a vol.All(ensure_list, <fail>)
```

Comment:
I opened https://github.com/alecthomas/voluptuous/pull/206 that will address the `TypeError` exception by `humanize_error`